docs/dev/connectors/pravega.zh.md

```diff
@@ -0,0 +1,418 @@
+{% endhighlight %}
+</div>
+</div>
+
+## Pravega Reader
+
+### Streaming API
+This connector provides a `FlinkPravegaReader` class to consume events from one or multiple Pravega stream in real-time.
+See the below for an example:
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+// Enable checkpoint to make state fault tolerant
+env.enableCheckpointing(...);
+
+// Define the Pravega configuration
+PravegaConfig config = PravegaConfig.fromParams(params);
+
+// Define the event deserializer
+DeserializationSchema<MyClass> deserializer = ...
+
+// Define the data stream
+FlinkPravegaReader<MyClass> pravegaSource = FlinkPravegaReader.<MyClass>builder()
+    .forStream(...)
+    .withPravegaConfig(config)
+    .withDeserializationSchema(deserializer)
+    .build();
+DataStream<MyClass> stream = env.addSource(pravegaSource);
+{% endhighlight %}
+</div>
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+val env = StreamExecutionEnvironment.getExecutionEnvironment
+
+// Enable checkpoint to make state fault tolerant
+env.enableCheckpointing(...)
+
+// Define the Pravega configuration
+val config = PravegaConfig.fromParams(params)
+
+// Define the event deserializer
+val deserializer = ...
+
+// Define the data stream
+val pravegaSource = FlinkPravegaReader
+    .builder[MyClass]()
+    .forStream(...)
+    .withPravegaConfig(config)
+    .withDeserializationSchema(deserializer)
+    .build()
+    
+val stream = env.addSource(pravegaSource)
+{% endhighlight %}
+</div>
+</div>
+
+### Batch API
+Similarly, a `FlinkPravegaInputFormat` class is offered for batch processing. It reads events of a stream as a `DataSet`.
+See the below for an example:
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+// Define the input format based on a Pravega stream
+FlinkPravegaInputFormat<EventType> inputFormat = FlinkPravegaInputFormat.<EventType>builder()
+    .forStream(...)
+    .withPravegaConfig(config)
+    .withDeserializationSchema(deserializer)
+    .build();
+
+DataSource<EventType> dataSet = env.createInput(inputFormat, TypeInformation.of(EventType.class))
+                                   .setParallelism(2);
+{% endhighlight %}
+</div>
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+// Define the input format based on a Pravega stream
+val inputFormat = FlinkPravegaInputFormat.builder[EventType]()
+    .forStream(...)
+    .withPravegaConfig(config)
+    .withDeserializationSchema(deserializer)
+    .build()
+val dataSet = env.createInput(inputFormat, TypeInformation.of(classOf[EventType])).setParallelism(2)
+{% endhighlight %}
+</div>
+</div>
+
+Details on more functionality such as checkpointing, optional start, metrics and
+watermark assigning can be found in the [document](http://pravega.io/docs/latest/connectors/connectors/index.html)
```

Comment:
broken URL and poorly named link

Is this the correct link? http://pravega.io/docs/latest/connectors/flink-connector/streaming/

Maybe link name is "Flink Streaming Connector document"?

Reply:
I have corrected and merged this link into the end of the introduction as a conclusion.